megamek/src/megamek/common/ASFBay.java

```diff
@@ -231,16 +244,16 @@ public void destroyDoor() {
             recoverySlots.remove(0);
         }
     }
-
+    
     // get doors should be different - first I must subtract the number of
-    // active recoveries
-    @Override
-    public int getDoors() {
+    // active recoveries  *** don't know that we need this anymore.
```

Comment:
If we don't need it, delete the code.  History exists because of git.

Reply:
Fixed.